setup.py

```diff
@@ -42,6 +42,7 @@
     "requests-mock==1.12.1",
     "scikit-learn==1.5.0",
     "statsmodels==0.14.2",
+    "time_machine==2.14.2",
```

Comment:
I suggest we remove freezegun, in this case. There is no point in having two dependencies that do the same(-ish) thing.

Reply:
https://github.com/oceanprotocol/pdr-backend/pull/1344